### ISSUE TITLE
Support for discord.py 2.0 features

### ISSUE THREAD

**Issue by super-cooper** [super-cooper/memebot#119]
We should integrate with new features, such as Threads, Events, etc. where applicable.

It looks like there is no full v2.0 changelog released, so the [migration guide](https://discordpy.readthedocs.io/en/stable/migrating.html) is probably the best reference to use. Desired changes should be listed in this issue.

Depends on #117

**Comment by super-cooper** [comment]
Now that the only relevant command is `/role`, this issue is no longer applicable. We may need some of those features for future work, in which case we'll keep the guide linked here in mind.
